alcs-api/src/application/application-decision/application-decision.controller.ts

```diff
@@ -43,28 +45,39 @@ export class ApplicationDecisionController {
 
   @Get('/application/:fileNumber')
   @UserRoles(...ANY_AUTH_ROLE)
-  async getAllForApplication(@Param('fileNumber') fileNumber): Promise<{
-    decisions: ApplicationDecisionDto[];
-    codes: ApplicationDecisionOutcomeTypeDto[];
-  }> {
+  async getByFileNumber(
+    @Param('fileNumber') fileNumber,
+  ): Promise<ApplicationDecisionDto[]> {
     const decisions = await this.appDecisionService.getByAppFileNumber(
       fileNumber,
     );
-    const codes = await this.appDecisionService.getCodeMapping();
-    const mappedDecisions = this.mapper.mapArray(
+    return this.mapper.mapArray(
       decisions,
       ApplicationDecision,
       ApplicationDecisionDto,
     );
-    const mappedCodes = this.mapper.mapArray(
-      codes,
-      ApplicationDecisionOutcomeType,
-      ApplicationDecisionOutcomeTypeDto,
-    );
+  }
 
+  @Get('/codes')
+  @UserRoles(...ANY_AUTH_ROLE)
+  async getCodes() {
+    const codes = await this.appDecisionService.fetchCodes();
     return {
-      decisions: mappedDecisions,
-      codes: mappedCodes,
+      outcomes: this.mapper.mapArray(
```

Comment:
not a required thing, but I prefer async mapping for arrays

Reply:
Done!